reverse_string.js

```diff
@@ -0,0 +1,12 @@
+function reverseString(s) {
+  let res = "";
+
+  for (let i = s.length - 1; i >= 0; i--) {
+    res = res + s[i]
+  }
+
+  return res;
+}
+
+console.log(reverseString("this is a string"))
+console.log(reverseString("aaron"))
```

Comment:
A few test cases you may want are an empty string, a string with a line break, and a string with special characters (eg unicode)